rh_ui/views/start.py

```diff
@@ -29,7 +29,7 @@ def start_post():
     uac = request.form.get('uac').upper().replace(' ', '')
     if error := pre_check_uac(uac):
         flash(error)
-        return redirect(url_for('i18n.start_bp.start_get'))
+        return render_template("start.html"), 422
```

Comment:
Would be good to get peoples opinions on the best status code to return. If you don't specify one it'll just return a `200` which doesn't seem right. From having a look around 422 Unprocessable Entity seems like the better one but I'm happy to change it because I'm not too sure

Reply:
Changed this to a 401 Unauthorized after some chats with Adam. Seems more appropriate